src/config/HYPRE_config.h.cmake.in

```diff
@@ -176,6 +176,9 @@
 /* As F77_FUNC, but for C identifiers containing underscores. */
 #define HYPRE_F77_FUNC_(name,NAME) name ## __
 
+/* Macro for silencing unused variable warnings */
+#define HYPRE_UNUSED_VAR(var) ((void) var)
```

Comment:
Is this the right place for the #define?  It's not configurable, so maybe it goes in `_hypre_utilities.h`?

Reply:
Yes, that makes sense! I'll move in there

Reply:
Done in https://github.com/hypre-space/hypre/pull/896/commits/9d5abd2f3f8fe3fea5c7d0f2aa27a7594b9dba19